### DISCUSSION TITLE
Space-x and flex-row-reverse

### DISCUSSION THREAD

**Discussion by RomainNeup** [tailwindlabs/tailwindcss#10498] [Ideas]
Hello,

Is it normal that the space-x-* is broken when used with flex-row-reverse
And same for space-y-* with flex-col-reverse ?

I understand why the problem happens but I feel like it's not supposed to be like that.

Thanks,
Romain

**Comment by wongjn** [comment]
Yep! That's why there is the `space-x-reverse` class for reversed horizontal layout or `space-y-reverse` for reversed vertical layouts, [see the documentation](https://tailwindcss.com/docs/space#reversing-children-order). 

Alternatively for flex layouts, you may wish to consider using [`gap`](https://tailwindcss.com/docs/gap).

**Reply by RomainNeup** [reply]
Oh, thanks a lot, I didn't read the documentation enough 😅

**Comment by mihir258** [comment]
Great information , really it help us a lot.
[Intel Dinar Chronicles
](https://apps.apple.com/vn/app/dinar-chronics/id1642043789)
